src/fidesapi/routes/util.py

```diff
@@ -17,3 +20,14 @@ def get_resource_type(router: APIRouter) -> str:
         The router's resource type
     """
     return router.prefix.replace(f"{API_PREFIX}/", "", 1)
+
+
+def obscure_string(plaintext_string: str) -> str:
+    "obscures a string as a minor security measure"
+
+    return b64e(zlib.compress(plaintext_string.encode(), 9)).decode()
```

Comment:
Is level-9 compression a requirement? It's somewhat arbitrary, but this function will run more quickly with a lower level, and the default (6) is a sound compromise.
```suggestion
    return b64e(zlib.compress(plaintext_string.encode())).decode()
```